database_connection_setup.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require './lib/database_connection'
```

Comment:
we set up something very similar. called setup_database.rb. We should ensure we only merge using one of these

Reply:
sure, we can discuss and decide which one to keep. Probably yours.